l2geth/consensus/clique/scheduler.go

```diff
@@ -216,6 +230,10 @@ func (schedulerInst *Scheduler) handleChainHeadEventLoop() {
 	for {
 		select {
 		case chainHead := <-schedulerInst.chainHeadCh:
+			if chainHead.Block.Transactions().Len() != 0 && chainHead.Block.Transactions()[0].GetMeta() != nil && chainHead.Block.Transactions()[0].QueueOrigin() == types.QueueOriginL1ToL2 {
+				log.Debug("chainHead", "block number", chainHead.Block.NumberU64(), "extra data", hex.EncodeToString(chainHead.Block.Extra()))
```

Comment:
`block number` -> `block_number`
`extra data` -> `extra_data`
Context name should not contains space character.
Besides, do you have realize that there will be only one chainHead event for sync batch blocks on a sequencer node. So when syncing a batch of blocks, only the chain head event for the last block will be emitted.